.github/workflows/run.brew-install-tests.yml

```
@@ -0,0 +1,47 @@
+name: "Tests: brew install"
+defaults:
+  run:
+    shell: bash
+
+env:
+  AWS_PROFILE: default
+  AWS_REGION: us-east-1
+
+on:
+  workflow_dispatch:
+#  schedule:
+#    - cron: '0 3 * * */2'
+
+jobs:
+  get_release_version:
+    name: Get Release Version
+    runs-on: ubuntu-latest
+    outputs:
+      release: ${{ steps.ize_release.outputs.release }}
+    steps:
+      - name: get ize version
+        id: ize_release_version
+        uses: rez0n/actions-github-release@main
+        env:
+          token: ${{ secrets.HOMEBREW_TAP_GITHUB_TOKEN }}
+          repository: "hazelops/ize"
+          type: "latest"
+
+      - name: Save release version
+        id: ize_release
+        run: |
+          echo "release=${{ steps.ize_release_version.outputs.release }}" >> $GITHUB_OUTPUT
+          echo ${{ steps.ize_release_version.outputs.release }}
+
+  install_ize:
+    name: Install On MacOS Latest
+    runs-on: macos-latest
+    needs: get_release_version
+    steps:
+      - name: Install Ize
+        run: |
+          brew tap hazelops/ize
+          brew install ize
+
+      - name: Run Ize and compare version
+        run: ize -version |grep ${{needs.get_release_version.outputs.release}}
```

I don't think it matters, but in our docs we use `--`. Also spaces add clarity.
```suggestion
        run: ize --version | grep ${{ needs.get_release_version.outputs.release }}
```